cypress/utils/utils.ts

```diff
@@ -1878,19 +1879,24 @@ export function validateMtaVersionInUI(expectedVersion: string): void {
     click(closeAbout);
 }
 
+interface NamedItem {
+    name: string;
+}
+
 /**
  * Takes 2 arrays of any type and returns array of elements, unique for a second array
  * @param arrA is an array of any type
  * @param arrB is an array of any type
  * @return result
  */
-export function getUniqueElementsFromSecondArray<T>(arrA: T[], arrB: T[]): T[] {
+export function getUniqueElementsFromSecondArray<T extends NamedItem>(arrA: T[], arrB: T[]): T[] {
```

Comment:
I don't think we need an interface that small just for this method, if we are using it just here we can do `T extends {name: string}`

Reply:
Done